.github/workflows/conformance.yml

```diff
@@ -0,0 +1,79 @@
+name: Manually run upstream conformance test
+
+on:
+  workflow_dispatch:
+    inputs:
+      antrea-version:
+        description: The Antrea version to test. It could be a SHA-1 value, a branch, or a tag (e.g. a7b012b, release-1.12, v1.12.0).
+        default: main
+        required: true
+      k8s-version:
+        description: K8s version (e.g. v1.27.1). Kind's default K8s version will be used if empty.
+        required: false
+      test-suite:
+        description: The test suite to run. Valid values include "whole-conformance", "conformance", "network-policy, "sig-network", and "all".
```

Comment:
Maybe add a comment to indicate 'all' is for both Conformance and Network Policy tests?

Reply:
It actually means "conformance", "network-policy, "sig-network". This just consumes run-k8s-e2e-tests.sh, I don't want to maintain redundant explanation here which would add maintenance efforts.
I added a pointer to the script in the latest commit.

Reply:
OK, we may refine the comment on the `run-k8s-e2e-tests.sh`, the comment of the parameter says `Run both Conformance and Network Policy tests.`, I thought only `conformance` and `network-policy` are included.

Reply:
corrected the script